src/view/dataElements/xdmObject/helpers/findSchemaMetaById.js

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export default ({ $id, schemasMeta }) => {
+  let foundSchemaMeta;
+  for (let i = 0; i < schemasMeta.length; i += 1) {
```

Comment:
Can't we use `schemasMeta.find(meta => meta.$id === $id)` ?

Reply:
Yes, that's a better implementation. I will change it